### DISCUSSION TITLE
Rules of PE and SIMD

### DISCUSSION THREAD

**Discussion by MFJI** [Xilinx/finn#1032] [Q&A] [answered]
Hello everyone,

I am interested to know whether there is a guide for the settings of PE and SIMD. Is there somewhere I can find all the rules how I can set PE and SIMD?

Thank you.

**Comment by auphelia** [comment, marked as answer]
Hi @MFJI ,
You can find more information in this Jupyter notebook: https://github.com/Xilinx/finn/blob/dev/notebooks/advanced/3_folding.ipynb and here is a table summarizing the possible settings: https://finn.readthedocs.io/en/latest/internals.html#constraints-to-folding-factors-per-layer
